src/widget/wlibrarytableview.cpp

```diff
@@ -100,33 +100,71 @@ void WLibraryTableView::moveSelection(int delta) {
             if (row + 1 < pModel->rowCount()) {
                 selectRow(row + 1);
             }
-
             delta--;
         } else {
             // i is negative, so we want to move the highlight up
             int row = current.row();
             if (row - 1 >= 0) {
                 selectRow(row - 1);
             }
-
             delta++;
         }
     }
 }
 
 void WLibraryTableView::saveVScrollBarPos(TrackModel* key){
+    qDebug()<<"";
+    qDebug()<<"    WLibraryTableView::saveVScrollBarPos";
+    qDebug()<<"       m_vScrollBarPosValues["<<key<<"] ="<<verticalScrollBar()->value();
     m_vScrollBarPosValues[key] = verticalScrollBar()->value();
+    if (currentIndex().isValid()) {
+        // Act only when there is a focused cell. For example, there is none
+        // when library features are added initially after start.
+        qDebug()<<"        QModelIndexList selection = selectionModel()->selectedRows();";
+        QModelIndexList selection = selectionModel()->selectedRows();
```

Comment:
currentIndex() was one of the first things I tried. This is the cell that was clicked last which might be a row you just removed from the selection and it might in a different place than the main selection. This doesn't really achieve the consistent solution I had in mind for a mixed workflow, mouse, keyboard & controller: keep selection AND scrollposition

good you're asking, I'll add this to the docu.